stable/gocd/README.md

```diff
@@ -349,6 +351,23 @@ The gocd service account can be associated with an existing role in the namespac
 helm install --namespace gocd --name gocd-app --set rbac.roleRef=ROLE_NAME stable/gocd
 ```
 
+#### Agent service account:
+
+Service account can be configured specifically for agents. This configuration also allows for the reuse of the top level service account that is used to configure the server pod. The various settings and their possible states are described below:
```

Comment:
Used to configure the server pod and kubernetes elastic agents.

Reply:
Not sure what you mean here.